src/leetCodeExecutor.ts

```diff
@@ -84,7 +85,7 @@ class LeetCodeExecutor implements Disposable {
     }
 
     public async signOut(): Promise<string> {
-        return await await this.executeCommandEx(this.nodeExecutable, [await this.getLeetCodeBinaryPath(), "user", "-L"]);
```

Comment:
It seems that its an old typo so I changed it too, am I right?

Reply:
yes